eth/v1alpha1/node.proto

```diff
@@ -62,47 +47,54 @@ service Node {
+    repeated string addresses = 4;
+    // Information about nodes to which this peer is connected.
+    repeated Peer peers = 5;
+    // The current epoch as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 current_epoch = 6;
+    // The current slot as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 current_slot = 7;
+    // The current block root as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 current_block_root = 8;
+    // The finalized epoch as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 finalized_epoch = 9;
+    // The finalized slot as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 finalized_slot = 10;
+    // The finalized block root as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 finalized_block_root = 11;
+    // The justified epoch as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 justified_epoch = 12;
+    // The justified slot as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 justified_slot = 13;
+    // The justified block root as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 justified_block_root = 14;
+    // The previous justified epoch as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 previous_justified_epoch = 15;
+    // The previous justified slot as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 previous_justified_slot = 16;
+    // The previous justified block root as far as the node is concerned; this is dependent on its current view of the chain.
+    uint64 previous_justified_block_root = 17;
 }
 
-// Information about the node version.
-message Version {
-    // A string that uniquely identifies the node and its version.
-    string version = 1;
-
-    // Additional metadata that the node would like to provide. This field may
-    // be used to list any meaningful data to the client.
-    string metadata = 2;
-}
-
-message ImplementedServices {
-    repeated string services = 1;
-}
-
-// Peers is a list of peer messages.
-message Peers {
-    repeated Peer peers = 1;
+enum SyncState {
+  // UNDEFINED means no status is available.
+  SYNC_UNKNOWN = 0;
+  // INACTIVE means the node is not currently syncing with its peers.
+  SYNC_INACTIVE = 1;
+  // CATCHUP means the node is in catchup mode; skipping full verification of information from peers in return for processing speed.
+  SYNC_CATCHUP = 2;
+  // ACTIVE means the node is actively syncing with its peers, carrying out full verification of information received.
+  SYNC_ACTIVE = 3;
```

Comment:
is this the normal state a node will be in? or SYNC_INACTIVE?

Reply:
The common state for a healthy node would be `SYNC_ACTIVE` _i.e._ actively syncing with its peers.  Perhaps a different word instead of active, for example "full", may clarify the state of the node regarding syncing?

Reply:
SYNC_INSYNC sounds like a band :P